vuepress/docs/next/tutorials/create/mfe/epc.md

```diff
@@ -0,0 +1,92 @@
+---
+sidebarDepth: 2
+---
+
+# Create an Entando Platform Capability 
+
+An Entando Platform Capability, or EPC, is a packaged capability in the form of a bundle that adds functionality to the platform such as additional UX controls in the App Builder. With an EPC, adding menu options, an API management page, or integrating a WCMS like Strapi from the App Builder is simplified. This tutorial demonstrates how to build an EPC from a simple React micro frontend bundle.
+
+## Prerequisites
+* A working instance of Entando
+* An existing [React MFE](react.md)
+
+
+## Convert the React Bundle into an EPC
```

Comment:
"the React Bundle into an EPC" -> "a React MFE into a simple EPC".  I'm not sure of the right language here. A PBC (or EPC) is composed of bundles. A bundle is composed of components, in this case a React MFE. It sounds wrong to say we're converting a React bundle to an EPC. Maybe "simple EPC" addresses the fact that this is the simplest possible EPC. A true EPC would likely have multiple components.